scripts/performance/measure.py

```diff
@@ -29,11 +29,10 @@ def pstdev(data):
     return (ss/n) ** 0.5
 
 
-def scenario(command):
+def scenario(command, loop=10):
```

Comment:
It's my intention to reduce the loop size to 10 now. The pstddev serves as an indicator of the stability of the performance test run. And because the test is associated with check in, there will be enough data collected.